Enhancement/UnlockForgeEnhancements.cs

```diff
@@ -268,7 +269,7 @@ public void ForgeWeaponEnhancement()
 
         LOC.Kitsune();
         Farm.Experience(30);
-        Farm.BlacksmithingREP(4, Bot.Config.Get<bool>("UseGold"));
+        Farm.BlacksmithingREP(4, Bot.Config.Get<bool>("UseGold"), CanSolo: Bot.Config.Get<bool>("CanSolo"));

```

Comment:
slugbutter isn't a hard boss to solo?

Reply:
its only an option the users can still use to do solo or pub room